src/Lean/Widget/UserWidget.lean

```diff
@@ -78,10 +78,7 @@ builtin_initialize moduleRegistry : ModuleRegistry ←
     toArrayFn     := fun es => es.toArray
   }
 
-/--
-Registers `[builtin_widget_module]` and `[widget_module]` and binds the latter's implementation
-(used for creating the obsolete `[widget]` alias below).
- -/
+/-- Registers a widget module. Its type must implement `Lean.Widget.ToModule`. -/
```

Comment:
Moved implementation detail below because the docstring is what's shown in hovers.